features/step_definitions/clear_data_on_signout_steps.rb

```diff
@@ -0,0 +1,22 @@
+Given(/^I am on the Money Manager tool$/) do
+  visit('/en/tools/money-manager/')
+end
+
+Then(/^I complete the series of questions$/) do
+  find_field(id: 'questionnaire_received_first_payment_true').click
```

Comment:
@igelstorm that's a good spot! Even if I disagree with the existing code of the frontend ... the "magic" place to look is on https://github.com/moneyadviceservice/frontend/blob/master/features/support/world/pages.rb